instrumentation-api/src/test/groovy/io/opentelemetry/instrumentation/api/db/SqlStatementSanitizerTest.groovy

```diff
@@ -105,6 +105,8 @@ class SqlStatementSanitizerTest extends Specification {
     sql                                                               | expected
     // Select
     'SELECT x, y, z FROM schema.table'                                | SqlStatementInfo.create(sql, 'SELECT', 'schema.table')
+    'SELECT x, y, z FROM `schema.table`'                              | SqlStatementInfo.create(sql, 'SELECT', 'schema.table')
```

Comment:
Wait, did we have backtick ` parsing?

Reply:
Yes, this already worked just didn't have a test.

Reply:
Nice 👍

Reply:
Actually it only works fine when you don't have a space inside backticks. If there is a space it returns the first word as the table name.

Reply:
Hm, we don't have any special rules for backticks, I guess they're just treated as "other characters" - that maybe that behavior is caused by this?

Reply:
yes, hopefully fixed now.